lib/stateTransition/createStateTransitionFactory.js

```diff
@@ -38,14 +41,25 @@ function createStateTransitionFactory(createDataContract) {
           return document;
         });
 
-        return new DocumentsStateTransition(documents);
+        stateTransition = new DocumentsStateTransition(documents);
+        break;
+      }
+      case types.IDENTITY_CREATE: {
+        stateTransition = new IdentityCreateTransition(rawStateTransition);
+        break;
       }
       case types.IDENTITY_CREATE: {
         return new IdentityCreateTransition(rawStateTransition);
       }
       default:
         throw new InvalidStateTransitionTypeError(rawStateTransition);
     }
+
+    stateTransition
```

Comment:
It should be set in the Abstract ST constructor.

Reply:
I thought about it, but how cat I do that? I mean, we pass `rawStateTransition` only to `IdentityCreateTransition`.

Reply:
Oh.. let's keep it as is and we will fix it when we refactor STs